### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by popeskul**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+version: "3.9"
+
+services:
+  postgres:
```
Comment (tarkovskynik) [inline docker-compose.yml:4]: ...-api:
build: ./
command: ./...-api
ports:
      - 8080:8080
    restart: on-failure
depends_on:
- ...-db
environment:
      - "DB_HOST=invoices-db"
      - "DB_PORT=5432"
      - "DB_USERNAME=postgres"
      - "DB_PASSWORD=qwerty"
      - "DB_NAME=postgres"
      - "SSL_MODE=disable"

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    image: postgres:latest
```
Comment (tarkovskynik) [inline docker-compose.yml:5]: db:
    restart: always
    image: postgres
    ports:
      - 5432:5432
    environment:
      - "POSTGRES_PASSWORD=qwerty"

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+FROM golang:1.18-alpine3.16 AS builder
+
```
Comment (tarkovskynik) [inline Dockerfile:2]: RUN go version
ENV GOPATH=/

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+WORKDIR /app
+COPY . .
+
```
Comment (tarkovskynik) [inline Dockerfile:5]: RUN go mod download

File: makefile [hunk 5]
```diff
@@ -12,6 +14,12 @@ migrate-up:
 migrate-down:
 	migrate -path ./schema -database 'postgres://$(db_user):$(db_password)@$(db_host):$(db_port)/$(db_name)?sslmode=disable' down -all
```
Comment (tarkovskynik) [inline makefile:16]: build: 
docker-compose build ...-api

run:
docker-compose up ...-api

**Comment by popeskul** [comment]
[Pavlo Popeskul](https://trello.com/c/6sGItd3w/72-pavlo-popeskul)

**Pull request merged**